config/environments/development.rb

```diff
@@ -32,4 +32,9 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.elasticsearch_client = Elasticsearch::Client.new(
+    host: 'localhost:9200',
+    log: true
```

Comment:
Need to support username/password since we are using a password-protected ES 5.x cluster.